shared/GIMessage.js

```diff
@@ -59,13 +81,31 @@ export class GIMessage {
     return instance
   }
 
+  decrypted (fn: ?Function): any {
```

Comment:
The name `decrypted` sounds like a boolean or boolean function to me, but it returns a decrypted op value (correct me if I am wrong); so what about `decryptedValue` ?

Reply:
OK, can make that change in a followup PR so as to not force another round of review!